src/api.js

```diff
@@ -173,3 +173,20 @@ export const MOCK_EVENT = {
 	visibility: 'public',
 	yes_rsvp_count: 23
 };
+
+// Mock category sourced from https://www.meetup.com/meetup_api/docs/find/topic_categories/
+export const MOCK_CATEGORY = {
```

Comment:
So this is just a single category pulled from `/find/topic_categories/`. I waffled on making this an array, which is what the source endpoint returns. As is, tests will likely need to do `[MOCK_CATEGORY]` for this to be useful or similarly build an array to desired length. Any thoughts, let me know.